cli/signal.rs

```diff
@@ -11,7 +21,33 @@ pub fn kill(pid: i32, signo: i32) -> Result<(), OpError> {
 }
 
 #[cfg(not(unix))]
-pub fn kill(_pid: i32, _signal: i32) -> Result<(), OpError> {
-  // TODO: implement this for windows
+pub fn kill(pid: i32, signal: i32) -> Result<(), OpError> {
+  unsafe {
+    match signal {
+      9 | 15 | 2 => {
+        let handle = if pid == 0 {
+          GetCurrentProcess()
+        } else {
+          OpenProcess(PROCESS_TERMINATE, 0, pid as DWORD)
```

Comment:
Check that pid is positive before casting to DWORD. If pid < 0, fail with "not supported".